src/figuresClasses.ts

```diff
@@ -1,11 +1,105 @@
-export interface Figure {}
+export type Shape = 'triangle' | 'circle' | 'rectangle';
+export type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
 
-export class Circle implements Figure {}
+  getArea(): number;
+}
+export class Triangle implements Figure {
+  shape: Shape;
+
+  color: Color;
+
+  private a: number;
+
+  private b: number;
+
+  private c: number;
+
+  constructor(color: Color, a: number, b: number, c: number) {
```

Comment:
```suggestion
  constructor(private color: Color, private a: number, private b: number, private c: number) {
```
You can do it directly in constructor